1-the-basics/style.css

```
@@ -0,0 +1,40 @@
+body {
```

prefer classes and avoid styling the <body> tag to prevent side effects. 

```suggestion
.main {
```

---

Yes, I need to correct this bad habit, thank you. 😅